mathesar_ui/src/component-library/collapsible/Collapsible.svelte

```diff
@@ -6,29 +6,32 @@
 
   export let isOpen = false;
   export let triggerAppearance: Appearance = 'default';
-
   function toggle() {
     isOpen = !isOpen;
   }
 </script>
 
 <div class="collapsible">
-  <slot name="trigger" {toggle} {isOpen}>
-    <Button
-      appearance={triggerAppearance}
-      class="collapsible-header"
-      on:click={toggle}
-    >
-      <Icon
-        {...iconExpandDown}
-        size="0.9rem"
-        rotate={isOpen ? undefined : 270}
-      />
-      <div class="collapsible-header-title">
-        <slot name="header" />
-      </div>
-    </Button>
-  </slot>
+  <div class="collapsible-header">
+    <slot name="trigger" {toggle} {isOpen}>
+      <Button
+        appearance={triggerAppearance}
+        class="collapsible-header-btn"
+        on:click={toggle}
+      >
+        <Icon
+          {...iconExpandDown}
+          size="0.9rem"
+          rotate={isOpen ? undefined : 270}
+        />
+        <div class="collapsible-header-title">
+          <slot name="header" />
+        </div>
+      </Button>
+      <slot name="trigger-aside" />
```

Comment:
The `trigger-aside` slot should be outside the `trigger` slot. This way the user can have a custom trigger as well as use the `trigger-aside` slot.

Reply:
Moved` trigger-aside` slot should be outside the `trigger` slot